cmd/server/flags.go

```diff
@@ -473,3 +473,14 @@ var flags = append([]cli.Flag{
 		Usage:   "Flag to decrypt all encrypted data and disable encryption on server",
 	},
 }, logger.GlobalLoggerFlags...)
+
+// If woodpecker is running inside a container the default value for
+// the datasource is different from running outside a container.
```

Comment:
Why they should differ?

Reply:
Only for backward compatibility.
Current working directory is not a sensible option for containers.